index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>portifolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="header">
+        <div class="style">
+          <div class="right">
+           <h2>9:41</h2>
+          </div>
+          <div class="left">
+           <img src="/images/Cellular Connection.png" alt="">
+           <img src="/images/Wifi.png" alt="">
+           <img src="/images/Battery.png" alt="">
+          </div>
+        </div>
+        <nav class="nav">
+         <h1>My logo</h1>
+        <div class="menu">
+          <img src="/images/Union1.png" alt="">
+        </div>
+        </nav>
+    </header>
+    <div class="header-desc">
+        <h2>Hey There.
+        I’m Antoh.</h2>
```

Comment:
- [ ] This is the main title of your portfolio, Therefore it would be a nice to use a `h1` tag. That will make easier to the search engine to search through your page and know about a specific topic, in this case, you. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.